core/domain/calculation_registry_test.py

```diff
@@ -0,0 +1,42 @@
+# coding: utf-8
+#
+# Copyright 2018 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Tests for calculation registry."""
+
+from core.domain import calculation_registry
+from core.tests import test_utils
+
+
+class CalculationRegistryTests(test_utils.GenericTestBase):
+    """Provides testing of the calculation registry."""
+
+    def test_calculation_registry(self):
+        """Do some sanity checks on the calculation registry."""
+        self.assertEqual(
+            len(calculation_registry.Registry.calculations_dict), 0)
+        calculation_registry.Registry.get_calculation_by_id('AnswerFrequencies')
```

Comment:
Hm. Actually ... drop this test, and make _calculations_dict private in calculation_registry.py (prefix with underscore). It probably shouldn't be exposed. No need to look at length of dict since we don't have a "public API" for that.

(Also double-check first that external callers aren't using calculations_dict directly -- which they shouldn't.)